modules/material-management/test/multi_keyring.test.ts

```diff
@@ -215,10 +215,10 @@ describe('MultiKeyring: onDecrypt', () => {
     const suite = new NodeAlgorithmSuite(AlgorithmSuiteIdentifier.ALG_AES128_GCM_IV12_TAG16)
     const unencryptedDataKey = new Uint8Array(suite.keyLengthBytes)
     const [edk0, keyringTrace0] = makeEDKandTrace(0)
-    const material = new NodeDecryptionMaterial(suite)
+    const material = new NodeDecryptionMaterial(suite, {})
 
     const generator = keyRingFactory({
-      async onDecrypt (material: NodeDecryptionMaterial /*, encryptedDataKeys: EncryptedDataKey[], context?: EncryptionContext */) {
+      async onDecrypt (material: NodeDecryptionMaterial /*, encryptedDataKeys: EncryptedDataKey[] */) {
```

Comment:
For me, this is similar to named parameters.  Ideally the _both_ the function declaration _and_ the call site should be descriptive of intention.

In this case, the function is "cut off" because I'm only using _some_ of the arguments.  My Typescript config is relatively strict and wants every defined variable to be used.  So I have to remove the arguments because they are not being used.  However when copying this code around or trying to understand what/how it is used, it is useful to see the full definition.

Reply:
Makes sense!